src/string/shortest_palindrome.rs

```diff
@@ -0,0 +1,48 @@
+pub fn shortest_palindrome(s: String) -> String {
+    if s.is_empty() {
+        return "String is Empty!".to_string();
```

Comment:
An empty string `""` is a palindrome. I think in this case an empty string should be returned:
```suggestion
        return "".to_string();
```